discord_thread_summarizer_using_openai/types.bal

```diff
@@ -0,0 +1,25 @@
+type ChannelThread record {
+    string guild_id;
+    string id;
+    string name;
+};
+
+type ActiveThreads record {
+    ChannelThread[] threads;
+};
+
+type Channel record {
+    string guild_id;
+    int 'type;
+};
+
+type Author record {
+    string username;
+};
+
+type Message record {
+    string content;
+    Author author;
+    string timestamp;
+    anydata edited_timestamp;
```

Comment:
Are we using this field anywhere?

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/69/commits/ca9ecd0b4475145df3519080000a28a26d049d7d